### DISCUSSION TITLE
how to create your own mirror?

### DISCUSSION THREAD

**Discussion by nightlytech** [termux/termux-packages#14292] [General]
via github pages

**Comment by sylirre** [comment]
> via github pages

Seriously, do you want to upload over 1 GiB to a repository on GitHub and serve from there? This is against GitHub ToS. Full clone of our apt repos is about 23 gigabytes.

We also already have a mirror that uses GitHub releases, a solution that doesn't violate terms of service. Making another mirror on same host is pointless and we will not add it to our lists.

If you want to help Termux project, please use a VPS or a VM in cloud. That solution would be paid but also reliable.
